### ISSUE TITLE
Bug with pagination bullet randomly disappears

### ISSUE THREAD

**Issue by dngraphisme** [nolimits4web/swiper#2843]
This is a :
* [x] bug

* Swiper Version: Swiper 3.4.2.
* Platform/Target and Browser Versions: MacOS, Chrome latest and Mozilla latest.
* Live Link or JSFiddle/Codepen or website with isssue: https://passport-oceans.com/collier.html

### What you did
This is my JS for the system >

`// main slides collier
$(document).ready(function () {
	if ($('body').is('.collier-page')) {
    var swiper = new Swiper('.swiper-container', {
        keyboardControl: true,
        grabCursor: false,
        allowTouchMove: false,
        touchRatio: 0,
        touchAngle:0,
        pagination: ".swiper-pagination",
        paginationBulletRender: function(swiper, index, className) {
            var name = document
                .querySelectorAll(".swiper-slide.h-slide")
                [index].getAttribute("data-name");
            return '<span class="' + className + '">' + name + "</span>";
        },
    });

    var vswiper1 = new Swiper('#slide-voyage', {
        scrollbar: '.swiper-scrollbar.scroll-voyage',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper2 = new Swiper('#slide-date', {
        scrollbar: '.swiper-scrollbar.scroll-date',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper3 = new Swiper('#slide-hebergement', {
        scrollbar: '.swiper-scrollbar.scroll-hebergement',
          /*scrollbar: {
		  	el: '.swiper-scrollbar.scroll-hebergement',
		  	hide: true,
      	},*/
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper4 = new Swiper('#slide-activite', {
        scrollbar: '.swiper-scrollbar.scroll-activite',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper5 = new Swiper('#slide-eco', {
        scrollbar: '.swiper-scrollbar.scroll-eco',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper6 = new Swiper('#slide-activitesupp', {
        scrollbar: '.swiper-scrollbar.scroll-activitesupp',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper7 = new Swiper('#slide-transport', {
        scrollbar: '.swiper-scrollbar.scroll-transport',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper8 = new Swiper('#slide-serviceplus', {
        scrollbar: '.swiper-scrollbar.scroll-serviceplus',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper9 = new Swiper('#slide-cadeau', {
        scrollbar: '.swiper-scrollbar.scroll-cadeau',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    var vswiper10 = new Swiper('#slide-assurance', {
        scrollbar: '.swiper-scrollbar.scroll-assurance',
        direction: 'vertical',
        slidesPerView: 'auto',
        mousewheelControl: true,
        freeMode: true
    });
    $('.go-step').click(function (event) {
        console.log(swiper);
        event.preventDefault();
        if ($(this).attr("data-next")) {
            var diapo = $(this).attr("data-next");
            console.log('clic next '+ diapo);
        }
        if ($(this).attr("data-prev")) {
            var diapo = $(this).attr("data-prev");
            console.log('clic prev '+ diapo);
        }
        swiper[0].slideTo(diapo-1, 200, true);

    });
    }

});`

### Expected Behavior
Get all bullets point of each slide

### Actual Behavior
Random loading of bullets points to the loading of the page

Thanks for your time and your API :)

**Comment by dngraphisme** [comment]
Hi validimir I am using swiper js for one of my clients but I am back to face a completely random bug on a page with pagination.

Here is the page > https://passport-oceans.com/collier.html

Once on 2 charge of it the swiper-pagination bullet disappear ... I do not understand where the problem is coming from.

If you want to know everything about how I mounted the system:

I created a global variable for a general swiper, then created 11 swiper to have the possibility to scroller inside the swiper-container.
